app/src/main/kotlin/com/wire/android/feature/auth/registration/CreateAccountActivity.kt

```diff
@@ -0,0 +1,20 @@
+package com.wire.android.feature.auth.registration
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import com.wire.android.R
+import kotlinx.android.synthetic.main.activity_create_account.*
+
+class CreateAccountActivity : AppCompatActivity(R.layout.activity_create_account) {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+//        replaceFragment(R.id.activityCreateAccountLayoutContainer, CreateAccountFragment.newInstance(), false)
+        initBackButton()
+    }
+
+    private fun initBackButton() {
+        activityCreateAccountBackButton.bringToFront()
```

Comment:
@android10  we are bringing the button to the front not the activity. If we don't call `bringToFront` this button will be hidden by the FrameLayout. @gizemb Don't remove this logic, maybe rename `activityCreateAccountBackButton` to `createAccountBackButton` to avoid any confusion. Rename the FrameLayout as well to `createAccountLayoutContainer`

Reply:
I solved this problem with elevation. It felt more natural to deal with it in the xml file rather than doing it programmatically.

Reply:
ok makes sense